include/graph.h

```diff
@@ -95,7 +101,13 @@ class Graph{
   std::atomic<uint64_t> num_updates;
 
   static Supernode* generate_delta_node(uint64_t node_n, long node_seed, uint64_t src,

```

Comment:
document please

Reply:
also, why are we both taking a `delta_loc` and returning a pointer?

Reply:
Will do. And I'll make it a void function